auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -104,71 +136,217 @@ import Vue from 'vue'
+    // Split string tokens into an array to avoid false string matching
+    const supportedEntityFlags = LaunchDarklyService.getFlag(LDFlags.IaSupportedEntities)?.split(' ') || []
+
+    return this.isNameRequest(business.corpType.code) &&
+      business.nameRequest?.enableIncorporation &&
+      supportedEntityFlags.includes(business.nameRequest?.legalType)
+  }
+
+  /** Returns the Name value for the affiliation */
+  private name (item: Business): string {
+    return this.isNumberedIncorporationApplication(item) ? 'Numbered Benefit Company' : item.name
+  }
+
+  /** Returns true if the Name Request is approved */
+  private isApprovedName (name: Names): boolean {
+    return name.state === NrState.APPROVED
+  }
+
+  /** Returns true if the Name Request is rejected */
+  private isRejectedName (name: Names): boolean {
+    return name.state === NrState.REJECTED
+  }
+
+  /** Returns the identifier for the affiliation */
+  private number (item: Business): string {
+    switch (true) {
+      // This case is subject to change once the NR Number is available in the Affiliation Response ALWAYS
+      case (this.isNumberedIncorporationApplication(item)):
+        return 'Pending'
+      case (this.isTemporaryBusinessRegistration(item.corpType.code)):
+        return 'Pending'
+      case this.isNameRequest(item.corpType.code):
+        return item.nameRequest.nrNumber
+      default:
+        return item.businessIdentifier
+    }
+  }
+
+  /** Returns the type of the affiliation */
+  private type (item: Business): AffiliationTypes {
+    switch (true) {
+      case (this.isNameRequest(item.corpType.code)):
+        return AffiliationTypes.NAME_REQUEST
+      case this.isTemporaryBusinessRegistration(item.corpType.code):
+        return AffiliationTypes.INCORPORATION_APPLICATION
+      default:
+        return AffiliationTypes.CORPORATION
+    }
+  }
+
+  /** Returns the corp full description for display */
+  private typeDescription (corpType: CorpTypeCd): string {
+    return GetCorpFullDescription(corpType)
+  }
+
+  /** Returns the status for the affiliation */
+  private status (item: Business): string {
+    switch (true) {
+      case (this.isNameRequest(item.corpType.code)):
+        // Format name request state value for Display
+        return item.nameRequest.state.charAt(0).toUpperCase() + item.nameRequest.state.slice(1).toLowerCase()
+      case this.isTemporaryBusinessRegistration(item.corpType.code):
+        return BusinessState.DRAFT
+      default:
+        return BusinessState.ACTIVE
+    }
+  }
+
+  /** Returns the folio number or a default message */
+  private folio (item: Business): string {
+    return item.folioNumber || this.$t('notAvailable').toString()
+  }
+
+  /** Returns the last modified date string or a default message */
+  private lastModified (item: Business): string {
+    return item.lastModified
+      ? new Date(item.lastModified).toLocaleDateString('en-CA', {
+        timeZone: 'America/Vancouver'
```

Comment:
will it be good to use with moment.js?

Reply:
Will look into this 👍

Reply:
Updated in next commit 👍

Reply:
Wait! Moment is obsolete. I recommend you look it up and updat this code accordingly (may want to create your own date mixin).

Also, JS time/date functions assume the user's browser/computer is set to the correct time/date, which can be faked (or just wrong). If time/date is used for decision-making then you shouldn't trust the user's time/date. This is a larger discussion - bring it up if necessary. I suspect, here, time/date is display only.

Reply:
This is for display only, you bet! I'm not sure this one display merits it's own Mixin in this ui though. 
I think i prefer to set it back to it how it was originally, i didn't realize Moment is no longer under development, although i'm not sure it's going anywhere anytime soon. 
Thanks for the heads up here!